README.md

```diff
@@ -75,6 +75,7 @@ The package exports a global `Avatar` object which has a property named `options
   - `cssClassPrefix`: This property is used to prefix the CSS classes of the DOM elements generated by this package.
   - `imageSizes`: This property defines the avatar sizes that are available to your project
   - `backgroundColor`: The background color for initials avatars (color name or hex value string). The default is gray (`"#AAA"`).
+  - `backgroundColorScheme`: Assign your own scheme for background colors for initials by passing a function here. The color scheme function should take only one argument, the `user` instance and return a CSS compatible color code e.g. `"#123abc"`.
```

Comment:
Hi. This is much better, however it now occurs to me (having seen this code) that it would be better to keep just the one `backgroundColor`option, and allow it to be **either** a string **or** a function. Having `backgroundColorScheme` override `backgroundColor` is an unnessesary complexity.

In the same way, the textColor option should also allow a function to be provided so that we can output a complimentary color to the background.